roles/openshift_facts/library/openshift_facts.py

```diff
@@ -317,9 +317,8 @@ def set_fluentd_facts_if_unset(facts):
 
     """
     if 'common' in facts:
-        deployment_type = facts['common']['deployment_type']
         if 'use_fluentd' not in facts['common']:
-            use_fluentd = True if deployment_type == 'online' else False
+            use_fluentd = False
```

Comment:
Unless I'm missing something, there should not be a change needed to openshift_facts, since this should only set the use_fluentd fact if it is not already set, which it should already be set by the change made in roles/openshift_common/tasks/main.yml.

I'm not against making False the general default, but @twiest or @wshearn could comment about changing the default for online deployments.